WIP feat(sabnzbd) adding exportarr for metrics

ac1dburnz opened the pull request:
**Description**
<!--
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
-->
⚒️ Fixes  # <!--(issue)-->

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-->

**📃 Notes:**
<!-- Please enter any other relevant information here -->

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
updates to sab

ac1dburnz committed:
Update questions.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update Chart.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Update values.yaml

wrong app 

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Merge branch 'master' into Sabnz

ac1dburnz committed:
Merge branch 'master' into Sabnz

Crow-Control commented on charts/stable/sabnzbd/values.yaml:
```diff
@@ -2,23 +2,90 @@ image:
+    replicas: 1
     podSpec:
       containers:
-        main:
```
why are ytou remving the main envs?

Crow-Control commented on charts/stable/sabnzbd/values.yaml:
```diff
@@ -2,23 +2,90 @@ image:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
```
We don't want you to run everything as root, unless its rquired.
Considering this ran without root before, this is not needed for the main container.

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz commented:
I added stuff from my repo by accident. I have corrected them

ac1dburnz committed:
Probs issues

This docker container doesn’t have probes that work. I always had to use a custom app on my own server. In order to have people use this app the health ports need to be disabled. 

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz commented:
@Ornias1993  i made the changes but also had to add one more. The probes dont work on this container, the app builds but i have never had it deploy on my system. I always had to manually disable them on my own catalog. I have disabled them.

ac1dburnz commented:
![IMG_1579](https://github.com/truecharts/charts/assets/10972668/de06d953-8c85-4171-8f96-edcf36bf0b5f)

Crow-Control commented on charts/stable/sabnzbd/values.yaml:
```diff
@@ -2,23 +2,93 @@ image:
+            PORT: "{{ .Values.service.metrics.ports.metrics.port }}"
+            URL: '{{ printf "http://%v:%v" (include "tc.v1.common.lib.chart.names.fullname" $) .Values.service.main.ports.main.port }}'
+            APIKEY: "{{ .Values.metrics.main.apiKey }}"
             SABNZBD__HOST_WHITELIST_ENTRIES: ""
```
you still moved this env var

ac1dburnz replied:
Hopeful what I did is what you meant ?

Crow-Control replied:
No.
LOOK CAREFULLY.

You removed this env var from the main container, without cause.
DO NOT make unrelated changes.

ac1dburnz replied:
@Ornias1993  hopeful that what you meant. If not, can you please explain? I'd love to learn what I did wrong here. I am still learning.

ac1dburnz replied:
or is it not suppose to be spit like that now and I need to move everything up

Crow-Control replied:
Every pod had and every container has it's own envs.

ac1dburnz replied:
ok good to know thanks

ac1dburnz committed:
Update values.yaml

ac1dburnz committed:
Merge branch 'master' into Sabnz

ac1dburnz committed:
Update values.yaml

Signed-off-by: Ac1dburn <ac1dburnez@gmail.com>

ac1dburnz committed:
Merge branch 'master' into Sabnz
